docs/sorting-algorithms/heap-sort.mdx

```diff
@@ -0,0 +1,234 @@
+---
+title: Heap Sort
+slug: /sorting-algorithms/heap-sort
+---
+
+# Introduction
+
+Heap Sort is a sorting algorithm which extracts the maximum or minimum element from the array and swap it with the last element of the array 
+for sorting in ascending or descending order N times.It is similar to Selection Sort, the only difference is that it uses Heap data structure 
+for finding minimum or maximum element. It is a In-Place and Non-Stable Sorting Algorithm.
+
+- Binary Heap: It is a complete binary tree represented using array.There are two types of Binary Heap:
+
+    - **MaxHeap**: Both left and right subtree of every node contains smaller elements
+    - **MinHeap**: Both left and right subtree of every node contains greater elements
+
+- Formulas:
+
+  - **Left Child**:     2*i+1
+  - **Right Child**:    2*i+2
+  - **Parent**:         floor((i-1)/2)
+
+`i` is the number of elements in the array.
```

Comment:
`Here i is the number....`


Keep `i` in backticks

Reply:
done